frappe/core/doctype/doctype/doctype.json

```diff
@@ -668,6 +669,13 @@
    "fieldname": "sender_name_field",
    "fieldtype": "Data",
    "label": "Sender Name Field"
+  },
+  {
+   "default": "0",
+   "description": "If enabled, Snapshot will be stored on every save, applicable only for draft mode.",
+   "fieldname": "allow_document_snapshots",
+   "fieldtype": "Check",
+   "label": "Allow Document Snapshots"
```

Comment:
```suggestion
   "label": "Allow Document Snapshots"
```

Kinda bad name for what it does. `Allow` doesn't fit here. 


Suggestion: `Enable Snapshots`


Change fieldname too.